packages/zoe/test/unitTests/contracts/test-simpleExchange.js

```diff
@@ -10,7 +10,7 @@ import { makeZoe } from '../../../src/zoe';
 import { setup } from '../setupBasicMints';
 import { setupNonFungible } from '../setupNonFungibleMints';
 import {
-  makeGetOfferHandle,
+  makeGetInviteHandle,
```

Comment:
I think we should be able to get rid of this method in this PR

Reply:
Resolving as a dup of above conversation